lib/converterbase.rb

```diff
@@ -934,14 +934,22 @@ def to_ruby(match, m1, m2, openclose_symbols)
       else
         # なろうのルビ対象文字を辿って｜を挿入する（青空文庫となろうのルビ仕様の差異吸収のため）
         # 空白もルビ対象文字に含むのはなろうの仕様である
-        m1.sub(/([#{CHARACTER_OF_RUBY} 　]+)$/) {
-          match_target = $1
-          if match_target =~ /^(　+)/
-            "#{$1}［＃ルビ用縦線］#{match_target[$1.length..-1]}"
+        if m2 =~ /^([ぁ-んァ-ヶーゝゞ・]+)[ 　]?([ぁ-んァ-ヶーゝゞ・]*)$/
+          f1, f2 = $1, $2
```

Comment:
__[rubocop]__
* Style/ParallelAssignment: Do not use parallel assignment. [:ref](https://github.com/bbatsov/ruby-style-guide#parallel-assignment)

<!-- Comment by [SideCI](https://sideci.com) -->